lib/fizzy/cxx20/span.hpp

```diff
@@ -4,7 +4,9 @@
 
 #pragma once
 
-#if __cplusplus > 201703L
+#ifdef __cpp_lib_span
+
+#warning "using c++20 span"
```

Comment:
@chfast So apparently this feature is not turned on in clang11?

Reply:
I don't know.

Reply:
Should we remove the C++20 implementation then?

Reply:
No.